db/rdbms/src/main/java/io/camunda/db/rdbms/write/service/SequenceFlowWriter.java

```diff
@@ -37,6 +37,16 @@ public void create(final SequenceFlowDbModel sequenceFlow) {
             sequenceFlow));
   }
 
+  public void insertIfNotExists(final SequenceFlowDbModel sequenceFlow) {
+    executionQueue.executeInQueue(
+        new QueueItem(
+            ContextType.SEQUENCE_FLOW,
+            WriteStatementType.INSERT,
```

Comment:
❓ What do we use this statement type for? Do we need to differentiate this from a regular INSERT somehow?

Reply:
Right now, the `WriteStatementType` is only used for sorting statements before sending them to MyBatis.

Reply:
Thank you 👍 We can probably treat them like the other INSERT statements then, I agree.